time timestamp

#0-1, where 1 is 100% load and 0 is completely idle.
float32 overall_cpu_load 
float32[] core_loads

#temperatures in degrees C.
float32 package_temp
float32[] core_temps